doc/source/actors.rst

```diff
@@ -21,96 +21,142 @@ Suppose we've already started Ray.
 Defining and creating an actor
 ------------------------------
 
-An actor can be defined as follows.
+Let's say we want to create an incrementing numerical Counter to use with Ray, 
+which returns its last return value + 1 each time it is called. However, the 
+Counter has to save some state-- its latest count value. 
 
-.. code-block:: python
+We can define this Counter as an actor by defining a ``Counter`` class, then 
+adding the ``@ray.remote`` decorator over it.
 
-  import gym
+.. code-block:: python
 
   @ray.remote
-  class GymEnvironment(object):
-    def __init__(self, name):
-      self.env = gym.make(name)
-      self.env.reset()
-    def step(self, action):
-      return self.env.step(action)
+  class Counter(object):
+    def __init__(self):
+      self.value = 0
+    def increment(self):
+      self.value += 1
+      return self.value
     def reset(self):
```

Comment:
It looks like we never use the `reset` method in this example, so maybe we can simplify the example by removing it?